apps/web/modules/settings/billing/components/BillingCredits.tsx

```
@@ -66,164 +70,182 @@ export default function BillingCredits() {
-          <Label>
-            {creditsData.credits.totalMonthlyCredits ? t("additional_credits") : t("available_credits")}
-          </Label>
-          <div className="mt-2 text-sm">{creditsData.credits.additionalCredits}</div>
-          <div className="-mx-6 mb-6 mt-6">
-            <hr className="border-subtle mb-3 mt-3" />
+  if (orgId && !isOrgScopedPath && !orgBranding?.slug) {
+    {
+      return null;
+    }
+
+    const { data: creditsData, isLoading } = trpc.viewer.credits.getAllCredits.useQuery({ teamId });
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)